src/Polyfill/Traits/AdminController/IsAnonymousAllowed.php

```diff
@@ -0,0 +1,28 @@
+<?php
```

Comment:
That files looked weird
The trait is the same + the function name is the same and it always return true

Reply:
This trait aims to fix the visibility of the method whether we are in PrestaShop >=9 or < 9